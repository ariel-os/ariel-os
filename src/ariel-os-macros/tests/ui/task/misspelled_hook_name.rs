#![no_main]

// FAIL: misspelled hook name
#[ariel_os::task(autostart, usb_builder_hooook)]
async fn main() {}
