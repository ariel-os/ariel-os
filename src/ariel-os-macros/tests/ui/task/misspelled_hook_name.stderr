error: unsupported parameter (`autostart`, `peripherals`, `pool_size`, and hooks `usb_builder_hook` are supported)
 --> tests/ui/task/misspelled_hook_name.rs:4:29
  |
4 | #[ariel_os::task(autostart, usb_builder_hooook)]
  |                             ^^^^^^^^^^^^^^^^^^
